### DISCUSSION TITLE
renderItem types?

### DISCUSSION THREAD

**Discussion by GollyJer** [Shopify/flash-list#542] [Q&A]
Hey! Testing out FlashList today and noticed it doesn't pass through the types in renderItem.

<img src=https://user-images.githubusercontent.com/689204/180917386-02c7ad48-fd56-4573-b3af-394edf4d830a.png width=580>

The values are properly typed when using `FlatList`. 
Is this expected, or am I doing something wrong?

**Comment by naqvitalha** [comment]
I'll check this and get back
